kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/java/org/kie/server/services/jbpm/ui/form/render/FormReader.java

```diff
@@ -35,7 +40,9 @@
     
     public FormReader() {
         this.mapper = new ObjectMapper();
-        this.mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+        JavaTimeModule module = new JavaTimeModule();
+        module.addSerializer(LocalDateTime.class, new LocalDateTimeSerializer(DateTimeFormatter.ofPattern("yyyy-MM-dd")));
```

Comment:
Would this show also hour part of the LocalDateTime? Isn't that also a supported data type and the time part should not be trimmed?

Reply:
In order to render the HTML properly the browsers (I checked Firefox and Chrome) requires that only the date part is present.

Reply:
Shouldn't this also add serializer also for LocalDate?

Reply:
Default LocalDate serializer, which just prints yyyy-MM-dd by default is enough.